docs/reference/remap/functions/parse_nginx_log.cue

```diff
@@ -0,0 +1,97 @@
+package metadata
+
+remap: functions: parse_nginx_log: {
+    category:    "Parse"
+    description: """
+        Parses Nginx access and error log lines. Lines can be in [`combined`](\(urls.nginx_combined)), or [`error`](\(urls.nginx_error)) format.
+        """
+    notices: [
+        """
+            Missing information in the log message may be indicated by `-`. These fields are omitted in the result.
+            """,
+    ]
+
+    arguments: [
+        {
+            name:        "value"
+            description: "The string to parse."
+            required:    true
+            type: ["string"]
+        },
+        {
+            name: "timestamp_format"
+            description: """
+                The [date/time format](https://docs.rs/chrono/latest/chrono/format/strftime/index.html) to use for
+                encoding the timestamp. The time is parsed in local time if the timestamp doesn't specify a timezone.
+                The default %d/%b/%Y:%T %z format for the combined logs and %Y/%m/%d %H:%M:%S for the error logs.
```

Comment:
For the error log, it's not that it is the default, but it is the only format that will parse since that is fixed in the regex.

I think my preference would be to change the regex if possible to parse any format to keep it consistent with the other types. It's annoying that the timestamp isn't properly delimited, but I think we could take any string up to the first `[` as the timestamp?

Reply:
So maybe we should only use the `timestamp_format` for the `combined` logs and ignore it for the `error` log.
I'll check for the timestamp ;)